hal/halx86/xbox/reboot.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * PROJECT:         Xbox HAL
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Xbox reboot functions
+ * COPYRIGHT:       Copyright 2004 Lehner Franz (franz@caos.at)
```

Comment:
Where does this come from?

Reply:
@HBelusca see first link in my comment above. :wink: